### ISSUE TITLE
Interactive Rebase: Include commit to rebase onto

### ISSUE THREAD

**Issue by Turbo87** [ForkIssues/Tracker#225]
In the interactive rebase dialog it's not always obvious in what order the commits are being applied. `git rebase -i` for example has an inverted order which might add to the confusion. As a solution to that, I would propose to include the commit that the branch is rebased onto in the list as a disabled line that can't be moved or dropped.

**Comment by DanPristupov** [comment]
Yes, I know, the default `rebase -i` list order is opposite, and this is confusing.

Showing the parent commit in the list won't solve the issue completely, but definitely will help. Thank you for the good suggestion!

**Comment by Turbo87** [comment]
the reason is most likely that it matches the order in which the commits will appear in the history graph of Fork

**Comment by DanPristupov** [comment]
@nebhale RI is a quite advanced feature for newbies and I tried to lower the barrier.

I spent a lot of time trying different UI approaches. I even erased nearly final code and started from scratch which took me additional week. One of the biggest problem which I had with IR myself is that after a long IR session I could not say if the result is correct because:
a. I edited list in apposite order and I forgot how the commit log had looked like
b. When RI was done, I saw the result and couldn't remember how graph had looked initially

I decided that allowing users to see the final result (i.e. same order as in commit list) is more important than representing how the feature works internally (original RI order). WYSIWYG approach, basically. It can be a *bit* confusing for pro users first time, but it's a big deal for newbies.

**Comment by nebhale** [comment]
Fair enough. 😄

**Comment by Turbo87** [comment]
@DanPristupov do you still think that this would be useful or should we close the issue? :)
